dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml

```diff
@@ -0,0 +1,69 @@
+:css
+  td {
+    padding-bottom: .5em;
+  }
+  td.heading {
+    padding-right: 1em;
+    text-align: right;
+    vertical-align: top;
+  }
+
+%p
+  We’re excited to continue to work together to bring computer science to your science classroom.
```

Comment:
No. Strange this was from the spec. I removed the word `science` before `classroom`